nntrainer/optimizers/optimizer_devel.h

```diff
@@ -88,25 +85,10 @@ class Optimizer : public ml::train::Optimizer {
     unknown = 7,
   };
 
-  /**
-   * @brief setProperty individually
-   * @note By passing empty string, this can validate if @a type is valid
-   * @param[in] key key to be passed as string
-   * @param[in] value value to be passed, if empty string is passed, do nothing
-   * but throws error when @a type is invalid
-   * @exception exception::not_supported     when string type is not valid for
-   * the particular layer
-   * @exception std::invalid_argument invalid argument
-   */
-  virtual void setProperty(const std::string &key,
-                           const std::string &value) = 0;
-
   /**
    * @brief     initialize optimizer.
-   * @retval #ML_ERROR_NONE Successful.
-   * @retval #ML_ERROR_INVALID_PARAMETER invalid parameter.
    */
-  virtual int initialize() = 0;
+  virtual void initialize(){};
```

Comment:
As we are making a api breaking change, may be finalize to keep modules consistent?

Reply:
will update.

Reply:
done.